docs/configuration/index.md

```diff
@@ -1737,6 +1737,16 @@ line.
 
 See [general query configuration](#general-query-configuration).
 
+###### Broker Generated Query Configuration Supplementation
+
+The Broker generates queries internally. This configuration section describes how an operator can augment the configuration
+of these queries.
```

Comment:
nit: I think an example here of why someone would want to set this functionality will be helpful to Druid operators.